acrobat/blocks/rnr/rnr.js

```diff
@@ -11,6 +11,7 @@ const { createTag } = await import(`${miloLibs}/utils/utils.js`);
 const COMMENTS_MAX_LENGTH = 500;
 const SHOW_COMMENTS_TRESHOLD = 5;
 const RNR_API_URL = 'https://rnr-stage.adobe.io/v1';
```

Comment:
Stage and prod should have the different API endpoints. We need to switch the endpoint based on environment.

Reply:
I was going to do it in a "cleanup" PR, this PR is only about the new API changes.